inspec/a2-iam-v2-integration/inspec.yml

```diff
@@ -4,8 +4,8 @@ maintainer: Chef Software, Inc.
 copyright: Chef Software, Inc.
 copyright_email: engineering@chef.io
 license: All Rights Reserved
-summary: An InSpec profile for integration testing the IAM v2 migration
+summary: An InSpec profile for integration testing the IAM v2.1 migration
```

Comment:
Man I keep going back and forth on this `v2` vs. `v2.1` issue 😅 I think since the test titles and controls all have `v2.1` in them, we should change the file names to `iam-v2p1...` for consistency

Reply:
I'll take the `p1` out of the buildkite jobs!